### ISSUE TITLE
Fix incorrect location for matching violating times in time-min-milliseconds

### ISSUE THREAD

**Issue by jeddy3** [stylelint/stylelint#4751]
> Clearly describe the bug

If there are two or more of the same violating times within a value they will all be reported at the location of the first violations.

We'll need to use the value parser to find the times so that we can make use of its `sourceIndex` property.

Ref: https://github.com/stylelint/stylelint/pull/4743

> Which rule, if any, is the bug related to?

`time-min-milliseconds`

> What code is needed to reproduce the bug?

```css
a { animation: foo 0.01s ease 0.8s, bar 0.8s ease 0.01s; }
```

> What stylelint configuration is needed to reproduce the bug?

```json
{
  "rules": {
    "time-min-milliseconds": "200ms"
  }
}
```

> Which version of stylelint are you using?

`13.3.3`

> How are you running stylelint: CLI, PostCSS plugin, Node.js API?

CLI with `stylelint "**/*.css"`

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

No.

> What did you expect to happen?

Warnings to be flagged at the correct locations.

> What actually happened (e.g. what warnings or errors did you get)?

Each warning is flagged at the first location.

**Comment by mattxwang** [comment]
I think I can address this in lockstep with adding end positions to the rule; going to assign myself!

**Comment by kawaguchi1102** [comment]
Hi @mattxwang .

Are you working on this issue ?
If not, I am ready to do this .

**Comment by mattxwang** [comment]
Ah - @kawaguchi1102, someone else ended up handling the end positions so I had not. Going to unassign myself, feel free to take this on!

**Comment by kawaguchi1102** [comment]
@mattxwang 
Thanks for the advice!

It seems that the problem has not been resolved even in the latest version.  
So I will work on this issue.
